EIPS/eip-3554.md

```diff
@@ -0,0 +1,59 @@
+---
+eip: 3554
+title: Difficulty Bomb Delay to December 1st 2021
+author: James Hancock (@madeoftin)
+discussions-to: https://github.com/ethereum/EIPs/issues/3554
+type: Standards Track
+category: Core
+status: Draft
+created: 2021-05-06
+---
+
+## Simple Summary
+Delays the difficulty bomb to show effect the first week of December 2021.
+
+## Abstract
+Starting with `FORK_BLOCK_NUMBER` the client will calculate the difficulty based on a fake block number suggesting to the client that the difficulty bomb is adjusting 9,500,000 blocks later than the actual block number.
+
+## Motivation
+Targeting for the Shanghai upgrde and/or the Merge to occur before Decemember 2021. Either the bomb can be readjusted at that time, or removed all together.
+
+## Specification
+#### Relax Difficulty with Fake Block Number
+For the purposes of `calc_difficulty`, simply replace the use of `block.number`, as used in the exponential ice age component, with the formula:
+
+    fake_block_number = max(0, block.number - 9_500_000) if block.number >= FORK_BLOCK_NUMBER else block.number
+
+## Rationale
+
+I choose this amount using the following script; credit @vbuterin with slight modifications by me to make it easier to use. The following predicts a 0.2 second delay to blocktime the first week of december and a 1 second delay by the end of the month. This gives reason to address because the effect will be seen, but not so much urgency we don't have space to work around if needed.
```

Comment:
Recommend removing casual discussion and fluff and focus purely on the standard.
```suggestion
The following script predicts a 0.2 second delay to blocktime the first week of december and a 1 second delay by the end of the month. This gives reason to address because the effect will be seen, but not so much urgency we don't have space to work around if needed.
```